templates/named.zone.erb

```diff
@@ -10,6 +10,9 @@ zone "<%= @zone %>" {
     file "<%= @zonefilename %>";
 <% end -%>
 <% if @zonetype == 'master' -%>
+<% if @forwarders.empty? && @master_empty_forwarders_enable -%>
+    forwarders {};
```

Comment:
Is there a reason to enforce that `forwarders` is empty? Isn't there a case to be made for `master_forwarders_enable`?

https://serverfault.com/questions/713256/purpose-of-a-forwarder-option-in-a-master-zone suggests there is also a case to be made to actually include forwarders, instead of discarding the `forwarders` values provided.

Reply:
Possibility to enforce empty forwarders is for disabling global forwarding when server is authoritative for domain and subdomain is delegated.
Related link:
https://kb.isc.org/docs/aa-00538

Possibility to include forwarders in master zone was introduced now as well.